src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/Algorithms/GenerateIPFColors.cpp

```diff
@@ -167,6 +167,7 @@ Result<> GenerateIPFColors::operator()()
   // Allow data-based parallelization
   ParallelDataAlgorithm dataAlg;
   dataAlg.setRange(0, totalPoints);
+  dataAlg.setParallelizationEnabled(false);
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Can we check if the DataArray is out-of-core then turn off parallel otherwise leave it on?